docs/hardware/backpack/hdzero-goggles.md

```diff
@@ -0,0 +1,47 @@
+---
+template: main.html
+description: How to setup, flash and update your HDZero Goggles ExpressLRS Backpack.
+---
+
+![Hardware-Banner](https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-Hardware/master/img/hardware.png)
+
+## Flashing the HDZero Goggles Backpack:
+
+After building the VRx firmware, copy the follwing 4 files into a folder named `ELRS` on the root of the SD card.
```

Comment:
*follwing

Can we elaborate where these 4 files are? "Build" would only show the 2 bin files currently, and none of the other files.
C:\.expresslrs-configurator\firmwares\github\Backpack\.pio\build\<target name>\ would usually have these files (at least 2 on current available targets).

Reply:
Spelling fixed.
I need to raise a ticket for configurator to copy the 4 files to the temporary folder that it opens.

Reply:
Created a ticket for the configurator guys https://github.com/ExpressLRS/ExpressLRS-Configurator/issues/479
Thats one app I can't get my head around to create a PR 🤪